CMakeLists.txt

```diff
@@ -162,7 +162,7 @@ if(DEFINED EXTENSIONS_DIRS)
     endforeach()
 endif()
 
-if(BABYLON_NATIVE_BUILD_APPS)
+if(BABYLON_NATIVE_BUILD_APPS OR ANDROID)
```

Comment:
I think it might be better if the Android CMakeLists.txt explicitly calls `add_subdirectory` of the Apps folder or just calls npm directly in the Android CMakeLists.txt? Thoughts?

Reply:
I've spent some time thinking about this and didn't find anything better. I don't want to have 2 calls to npm because DRY. another call to add_subdirectory seems weird because there is a call just before on the parent directory. so, I would keep it like that for now with a comment.

Reply:
This is going to be bad for someone trying to consume Babylon Native from an Android project that doesn't care about apps (say BRN). They will not be able to explicitly specify `BABYLON_NATIVE_BUILD_APPS=OFF`.

Reply:
You are right. what about a cmake function defined in root cmakelists and called in apps\cmakelists and in android app cmake?

Reply:
I'm not sure. Doesn't seem much better than setting `BABYLON_NATIVE_BUILD_APPS=ON` to me.

Reply:
Works for me.